### DISCUSSION TITLE
请问该项目是开源的吗，我想在这个软件的基础上增加命令行启动的功能

### DISCUSSION THREAD

**Discussion by whp98** [KtzeAbyss/Easy-Virtual-Display#10] [Q&A] [answered]
我想的是如何在这个项目的基础上增加一个命令行启动的方式，因为有的电脑需要重启（拔掉显示器），如果重启之后分辨率异常想要开启就会比较麻烦，我看了下本仓库，里面那个c++文件似乎不是我从release中下载的，所以想要自行添加功能无处发力，来这问一下。

**Comment by KtzeAbyss** [comment, marked as answer]
核心代码完全开源 请参见parsec-vdd项目
本项目首页已提供跳转链接，可自行进行c++修改编译
